bundles/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/OneWireBinding.java

```diff
@@ -78,14 +79,6 @@ public void deactivate() {
         ivOneWireReaderScheduler.stop();
     }
 
-    protected void addBindingProvider(OneWireBindingProvider bindingProvider) {
-        super.addBindingProvider(bindingProvider);
-    }
-
-    protected void removeBindingProvider(OneWireBindingProvider bindingProvider) {
-        super.removeBindingProvider(bindingProvider);
-    }
-
```

Comment:
revert deleting this methods. They are necessary to make the binding work within the karaf container (oh2)